### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by kdenhartog**

### CODE REVIEW

**Review by fallaciousreasoning — COMMENTED** [review]
This LGTM - just checking by "the current stable release version number (the version downloadable on https://brave.com/download)" we mean the version we're using in brave/brave-core?

**Comment by kdenhartog** [comment]
Ahh good point, I just copied the brave-browser one but that wording probably isn't correct for this repo. Let me reword.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+All versions including and above the current stable release version number (the version downloadable on https://brave.com/download).
```
Comment (kdenhartog) [inline SECURITY.md:5]:
```suggestion
Any version of adblock-rust used in the current stable release version or above of Brave Browser, or in any other repo under https://github.com/brave. This includes [brave-core](https://github.com/brave/brave-core) and [brave-ios](https://github.com/brave/brave-ios).
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Any version of adblock-rust used in the current stable release version or above of Brave Browser, or in any other repo under https://github.com/brave. This includes [brave-core](https://github.com/brave/brave-core) and [brave-ios](https://github.com/brave/brave-ios).
```
Comment (fallaciousreasoning) [inline SECURITY.md:5]:
```suggestion
Any version of Leo used in the current stable release version or above of Brave Browser, or in any other repo under https://github.com/brave. This includes [brave-core](https://github.com/brave/brave-core) and [brave-ios](https://github.com/brave/brave-ios).
```

**Review by fallaciousreasoning — APPROVED** [review]
LGTM

**Pull request merged**